src/app/components/RawRoute.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
I'm thinking it might make more sense to either only render Raw Test Routes or possibly use a key to indicate raw routes are desired. at this point, only tests seems fine

Reply:
I'd prefer to just allow everything. I'd like remove special logic around the test routes.

Reply:
Did you choose to render each route here for the 404 result instead of the site redirect?

Reply:
I honestly think we shouldn't redirect on error. If this was a test page, i'd rather see a 404 in a screenshot than the home page. It'd be more obvious what the problem was.

Reply:
I'd like to change the component to also render a 404 if the route isn't found.